src/coreComponents/common/LogLevelsInfo.hpp

```diff
@@ -0,0 +1,171 @@
+/*
+ * ------------------------------------------------------------------------------------------------------------
+ * SPDX-License-Identifier: LGPL-2.1-only
+ *
+ * Copyright (c) 2018-2020 Lawrence Livermore National Security LLC
+ * Copyright (c) 2018-2020 The Board of Trustees of the Leland Stanford Junior University
+ * Copyright (c) 2018-2020 TotalEnergies
+ * Copyright (c) 2019-     GEOSX Contributors
+ * All rights reserved
+ *
+ * See top level LICENSE, COPYRIGHT, CONTRIBUTORS, NOTICE, and ACKNOWLEDGEMENTS files for details.
+ * ------------------------------------------------------------------------------------------------------------
+ */
+
+/**
+ * @file LogLevelsInfo.hpp
```

Comment:
Please describe the responsability of this file here (i.e. as it is done in DataTypes.hpp).
I would expect this file to be a repository of all common `LogLevelInfo.hpp` + to expose the way to use them.

Also, should this file be located in `dataRepository`, or is there futures potencial uses in `common` or `codingUtilities`?

Reply:
DataRepo could be the right place, for now, no uses in common but it's a little unclear whether the package `dataRepository` should contain LogLevelInfo
For now I will move the file `LogLevelsInfo.hpp` into common/logger